run.sh

```diff
@@ -38,18 +76,24 @@ if [[ "x$1" != "x--restart" ]]; then
   fi
 fi
 
-pushd .
-  # Ensure rustc is updated
-  echo "* Updating rust if necessary..."
+pushd . &> /dev/null
   cd runtime
-  multirust override nightly-2015-08-10
 
-  # Compile runtime server
-  echo "* Compiling server... (This takes a while)"
+  # Ensure rustc is updated.
+  multirust list-toolchains | grep "$rustVersion" &> /dev/null
```

Comment:
@joshuafcole Cool. Can you elaborate more on the case that could be more consistent?